Modules/IO/NRRD/test/itkNrrdImageIOTest.h

```diff
@@ -153,6 +168,42 @@ itkNrrdImageIOTestReadWriteTest(std::string fn, unsigned int size, std::string i
       return EXIT_FAILURE;
     }
   }
+  if (inputFile == "null")
+  {
+    // check interporated metadata
```

Comment:
`interporated`? Maybe: Interpreted? Incorporated? Interpolated?

Reply:
should be incorporated